.goreleaser.yaml

```diff
@@ -0,0 +1,62 @@
+# TODO @JPERRY REMOVE THESE LINES AND ENSURE THE 'GITHUB_TOKEN' token is populated in the repo.
+# .goreleaser.yaml
+env_files:
+  github_token: ~/.path/to/my/github_token
+
+
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+
+universal_binaries:
+  - replace: false
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    dir: src
+    ldflags:
+      - -s -w -X github.com/defenseunicorns/zarf/src/config.CLIVersion={{.Version}}
+    goarch:
+      - amd64
+      - arm64
+archives:
+  -
+    format: binary
+    replacements:
+      darwin: Darwin
+      linux: Linux
+
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  use: github-native
+
+release:
+  github:
+    owner: defenseunicorns
+    name: zarf
+  prerelease: auto
+  mode: append
+
+brews:
```

Comment:
So if I'm reading this right, this is what will bump a new version on that repo, correct?

Reply:
This will replace the recipe in our tap repository so that `brew install zarf` will always install the latest release of zarf.
The user will probably have to do a `brew update` depending on the last time they naturally ran that.

Reply:
This will replace the recipe in our tap repository so that `brew install zarf` will always install the latest release of zarf.
The user will probably have to do a `brew update` depending on the last time they naturally ran that.